sushy_oem_idrac/utils.py

```diff
@@ -38,3 +43,32 @@ def reboot_system(system):
         system.refresh()
 
     LOG.info('System powered ON')
+
+@retrying.retry(
+    retry_on_exception=lambda exception: isinstance(exception, Exception),
+    stop_max_attempt_number=_IDRAC_IS_READY_RETRIES,
+    wait_fixed=_IDRAC_IS_READY_RETRY_DELAY_SEC * 1000)
+def wait_for_idrac_ready(oem_manager):
+    is_ready_response = is_idrac_ready(oem_manager)
```

Comment:
Let's move the check that determines if the iDRAC is ready into the is_idrac_ready method, and have that return True or False.

Reply:
Addressed.